src/app/layout.tsx

```diff
@@ -28,11 +31,17 @@ export default function RootLayout({
       <body
         className={`${geistSans.variable} ${geistMono.variable} antialiased`}
       >
-        <main className={`${geistSans.className} h-screen w-screen flex flex-col justify-center items-center bg-gradient-to-tr from-indigo-900 to-purple-600`}>
```

Comment:
i think its better to give the css positioning to each page's layout, so that if we want to change the design on some page it will be easier to do so.

Reply:
What I meant was, we can give each page, the positioning. Ik it will be repetitive but in case we want to adjust the position of a particular page, it will be easier to do so. 
If that can be countered somehow, then I dont mind using the css in main itself.

Reply:
oh, each page will have its own positioning, this is just meant to allot a certain amount of space to navbar children and footer in the layout, now that you say that tho i think it would be better to use min-h-screen over h-screen